integration-tests/src/test/java/oracle/kubernetes/operator/utils/LoadBalancer.java

```diff
@@ -190,7 +190,12 @@ private String getKubernetesNamespaceToUpdate(String domainNamespace) throws Exc
         String trimmed = debracketed.replaceAll("\\s+", "");
         // now have an ArrayList containing "foo", "bar" and "baz"
         ArrayList<String> list = new ArrayList<String>(Arrays.asList(trimmed.split(",")));
-        list.add(domainNamespace);
+        if (!list.contains(domainNamespace)) {
```

Comment:
You could just make the list a set.

Reply:
This file was not part of my PR. I don't want to make changes to this file without running all the tests as it is a common file used by all the tests - worried about breaking tests. I can maybe take it up as separate work to fix this after discussing with Vanaja.